src/internal/connector/graph/api/api.go

```diff
@@ -24,6 +24,12 @@ func NextLink(pl PageLinker) string {
 	return ptr.Val(pl.GetOdataNextLink())
 }
 
-func NextAndDeltaLink(pl DeltaPageLinker) (string, string) {
-	return NextLink(pl), ptr.Val(pl.GetOdataDeltaLink())
+func NextAndDeltaLink(pl PageLinker) (string, string) {
+	dpl, ok := pl.(DeltaPageLinker)
```

Comment:
I'm not sure we can completely comply with that in the graph package without jumping through a lot of hoops. In some places we probably could, but in others, due to the way graph is written with lots of different types, it may not be possible while retaining the common code we have already

Reply:
I'm assuming https://github.com/alcionai/corso/pull/3212/commits/0b945758572967ee4414028710edcaa301ca6d17 is what you meant.

Reply:
yes, just the PageLinker/DeltaPageLinker in this case.